.circleci/config.yml

```diff
@@ -45,13 +45,24 @@ jobs:
           name: Deploy to gcloud test project
           working_directory: ~/workbench/api
           command: |
-            ./project.rb circle-deploy \
-              --project all-of-us-workbench-test --creds-file circle-sa-key.json
+            if [ "${CIRCLE_BRANCH}" == "master" ]; then
```

Comment:
While you're in here: can you change things so the dependency check analysis only runs for the staging push, and not for test / other pushes?

It's super-flaky and annoying to have it run on every push / merge.

Reply:
Done.